code/game/objects/items/melee/f13powerfist.dm

```diff
@@ -156,31 +156,68 @@
 /obj/item/twohanded/chainsaw
 	name = "chainsaw"
 	desc = "A versatile power tool. Useful for limbing trees and delimbing humans."
-	icon_state = "chainsaw_off"
-	lefthand_file = 'icons/mob/inhands/weapons/chainsaw_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/weapons/chainsaw_righthand.dmi'
-	flags_1 = CONDUCT_1
-	force = 13
+	icon = 'icons/fallout/objects/melee/melee.dmi'
+	lefthand_file = 'icons/fallout/onmob/weapons/melee2h_lefthand.dmi'
+	righthand_file = 'icons/fallout/onmob/weapons/melee2h_righthand.dmi'
+	icon_state = "chainsaw"
+	item_state = "chainsaw"
+	var/on_icon_state = "chainsaw_on"
+	var/off_icon_state = "chainsaw"
+	var/on_item_state = "chainsaw_on"
+	var/off_item_state = "chainsaw"
```

Comment:
Please reorganize the icons.

Put the redefined ones (the ones that don't start with `var/`) first, and the newly defined down down.

Don't think there's anything here of that type, but if there's any reference to an object, leave it at the bottom so it's easily visible and taken into account to prevent hard deletes.